SynCommons.pas

```diff
@@ -55886,10 +55886,8 @@ procedure TextColor(Color: TConsoleColor);
     exit;
   TextAttr := ord(color);
   if ord(color)>=8 then
-    write(#27'[1;3') else
-    write(#27'[0;3');
-  write(AnsiTbl[(ord(color) and 7)+1],'m');
-  ioresult;

```

Comment:
why did you remove ioresult call ? IIRC it doesn't call any sys API.

Reply:
This is my mistake. I think call to ioresult it do nothing in this case, but now I understand it set global `InOutRes` variable to 0.